### PR TITLE
Feature/map menu

### PR DESCRIPTION
**Pull request by benkt2**

### CODE REVIEW

File: app/controllers/api/map_controller.rb [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+                                layer_type: 'raster',
+                                mapbox_tileset: 'unepwcmc.ahwxct9v'
+                              }, {
+                                id: 2,
+                                name: 'Species',
+                                information: 'abc',
+                                legend: ['rgb(254, 240, 217)', 'rgb(253, 204, 138)', 'rgb(252, 141, 89)', 'rgb(227, 74, 51)', 'rgb(179, 0, 0)'],
+                                layer_type: 'vector',
+                                carto_username: 'carbon-tool',
+                                carto_api_key: 'f7762e628586b3ff41a371b8e89ea0069e975299',
```
Comment (spencerldixon) [inline app/controllers/api/map_controller.rb]: ⚠️ Committing secrets!

File: app/controllers/api/map_controller.rb [hunk 2]
```diff
@@ -0,0 +1,51 @@
+class Api::MapController < ActionController::API
+  before_action :set_headers
+
+  def set_headers
+    response.headers['Content-Type'] = 'application/json'
+  end
+
+
+  def layers
+    render json: [{
```
Comment (spencerldixon) [inline app/controllers/api/map_controller.rb]: Perhaps we can abstract this json stub into a file that we can load in here as a dummy response, would help keep things clean

File: app/controllers/api/map_controller.rb (hunk 1 above)
Comment (benkt2) [reply app/controllers/api/map_controller.rb]: Put in env or use the secrets thingamy?

Comment (spencerldixon) [reply app/controllers/api/map_controller.rb]: Encrypted secrets preferrably, but let scrap this key and regenerate a new one too. Especially if it's being used in IBAT

Comment (spencerldixon) [reply app/controllers/api/map_controller.rb]: `EDITOR=vim rails credentials:edit` the trick is to use a real text editor 😉

Comment (benkt2) [reply app/controllers/api/map_controller.rb]: That's just mean 👎

**Comment by benkt2** [comment]
This PR is now defunct as the map functionality has changed completely!

**Pull request closed without merging**